src/scenes/gamescreen.ts

```diff
@@ -75,21 +72,24 @@ export class GameScreen extends Phaser.Scene {
     }
 
     private displayPlacedTiles(): void {
-        assert(this.placedTilesContainer);
+        const placedTiles: Nullable<Tile[]> = TileManager.getPlaced();
+        assert(this.placedTilesContainer && placedTiles);
         this.placedTilesContainer.removeAll(true);
 
-        const placedTiles: Nullable<Tile[]> = TileManager.getPlaced();
-        assert(placedTiles);
-        placedTiles.forEach((tile: Tile) => {
-            assert(tile.coordinateX && tile.coordinateY);
-            assert(this.placedTilesContainer);
-            this.placedTilesContainer.add(
-                this.add.image(
-                    tile.coordinateX * GameScreen.tilePixels,
-                    tile.coordinateY * GameScreen.tilePixels,
-                    `tile${tile.type}`,
-                ),
-            );
+        const startingTiles: Tile[] = TileManager.getStartingTiles();
+        const allTiles: Tile[] = [...placedTiles, ...startingTiles];
+        
+        allTiles.forEach((tile: Tile) => {
+            if (tile.coordinateX !== null && tile.coordinateY !== null) {
```

Comment:
Interesting change with this if instead of the assert... what led to this?

Reply:
if it holds up its a good practice to flip the if around instead of checking for not null, check for null and if so return & but the code after the if.

Reply:
if tile.coordinateX has the value 0 of type int, assert(tile.coordinateX) will throw exception

Reply:
@paul891112 absolutely, thats why you have to do the assert like this when dealing with int: `assert(x !== null)`

Reply:
+1